### DISCUSSION TITLE
SVG lettericons

### DISCUSSION THREAD

**Discussion by gurgeous** [mat/besticon#54] [Ideas]
I was messing around with SVG lettericons this morning. Which font family do we like the best? Here is the SVG:

```svg
<svg width="100" height="100" xmlns="http://www.w3.org/2000/svg">
  <rect x="0" y="0" width="100" height="100" fill="#909090"/>
  <text
    x="50%" y="50%" dy="0.10em"
    font-family="Helvetica Neue, Helvetica, sans-serif" font-size="75"
    dominant-baseline="middle" text-anchor="middle"
    fill="white">B</text>
</svg>
```

![image](https://user-images.githubusercontent.com/526195/109329490-7bb16d00-780f-11eb-9679-384698d0cc1e.png)

**Comment by mat** [comment]
Nice 😃 I think I like `Helvetica Neue,...` best. Disclaimer: Not a designer. Which one do you prefer?

**Reply by gurgeous** [reply]
Excellent, that would be my pick too! I'll go ahead and add lettericon svg support using this design. As per the earlier PR, the server will only return an SVG if `format=` contains `svg`.
